examples/fi_example.py

```diff
@@ -52,63 +51,73 @@ def main():
                     "Effective Duration",
                     "Effective Convexity"]
 
-    security1 = FISecurity("Price", 100.285, 10000.0, "912828ZG8", "20201202", "UST")
-    security2 = FISecurity("Price", 101.138, 200000.0, "US037833AR12", "20201203", "UST")
-
-    securities = [security1, security2]
-
-    jobSettings = FIJobSettings("20201201")
-
+    securities = [
+        FISecurity(calc_from_method="Price",
+                   calc_from_value=100.285,
+                   symbol="912828ZG8",
+                   settlement="20201202",
+                   discount_curve="UST",
+                   face=10000.0),
+        FISecurity(calc_from_method="Price",
+                   calc_from_value=101.138,
+                   symbol="US037833AR12",
+                   settlement="20201203",
+                   discount_curve="UST",
+                   face=200000.0)
+    ]
+    jobSettings = FIJobSettings(as_of_date="20201201")
     fi_calculation_parameters = FICalculationParameters(securities, calculations, jobSettings)
+    fi_calculation_parameters_root = FICalculationParametersRoot(data=fi_calculation_parameters)
 
-    print(fi_calculation_parameters)
+    print("FI Calculation Parameters Root:")
+    print(fi_calculation_parameters_root)
 
     fi_calculations_api = FICalculationsApi(api_client)
-    run_calculation_response = fi_calculations_api.run_fi_calculation_with_http_info(
-        fi_calculation_parameters=fi_calculation_parameters)
+    run_calculation_response = fi_calculations_api.post_and_calculate(
+        fi_calculation_parameters_root=fi_calculation_parameters_root,
+        _return_http_data_only=False)
 
     if run_calculation_response[1] != 202 and run_calculation_response[1] != 201:
         print_error(run_calculation_response)
         sys.exit()
 
     if run_calculation_response[1] == 201:
-        print_result(run_calculation_response[0])
+        output_calculation_result(run_calculation_response[0])
         sys.exit()
 
-    calculation_id = run_calculation_response[2].get("location").split("/")[-1]
+    calculation_id = run_calculation_response[0].data.id
```

Comment:
Shouldn't it be `run_calculation_response[0].data.calculationid`?

Reply:
It didn't display that behavior when I ran and tested, is that a problem?

Reply:
No, because the model that gets generated `CalculationInfo` has that field being `id` https://github.com/factset/analyticsapi-engines-python-sdk/blob/140b36ca933e15a4887b37565668bf2a72d842e1/auto-generated-sdk/fds/analyticsapi/engines/model/calculation_info.py#L85